### PR TITLE
IMAGINGCLOUD-322

### PR DESCRIPTION
**Pull request by s-zubov**

### CODE REVIEW

File: test/api_tester.py [hunk 1]
```diff
@@ -45,12 +45,12 @@ class ApiTester(unittest.TestCase):
...
     def setUp(self):
         self.failed_any_test = False
-        self.default_storage = 'Local-CI'
+        self.default_storage = 'Local-CI-Linux'
         self.cloud_test_folder_prefix = 'ImagingCloudTestPython'
         self.original_data_folder = 'ImagingIntegrationTestData'
         self._server_access_file = 'serverAccess.json'
         self._api_version = 'v3.0'
-        self._base_url = 'https://api.aspose.cloud/'
+        self._base_url = 'http://localhost:57972'
```
Comment (alexermakoff) [inline test/api_tester.py]: Is it correct URL?

File: test/api_tester.py [hunk 2]
```diff
@@ -45,12 +45,12 @@ class ApiTester(unittest.TestCase):
 
     def setUp(self):
         self.failed_any_test = False
-        self.default_storage = 'Local-CI'
+        self.default_storage = 'Local-CI-Linux'
```
Comment (alexermakoff) [inline test/api_tester.py]: Is it correct storage?

File: test/api_tester.py (hunk 1 above)
Comment (s-zubov) [reply test/api_tester.py]: Local debug info slipped into the commit. Fixed

File: test/api_tester.py (hunk 2 above)
Comment (s-zubov) [reply test/api_tester.py]: Local debug info slipped into the commit. Fixed

**Pull request merged**